distribution/lib/Standard/Table/0.0.0-dev/package.yaml

```diff
@@ -15,7 +15,7 @@ component-groups:
           - Standard.Table.Data.Table.new
           - Standard.Table.Data.Table.from_rows
           - Standard.Table.Data.Column.from_vector
-          - Standard.Table.Io.Csv.from_csv
+          - Standard.Base.System.File.read
```

Comment:
@jdunkerley I had to remove `from_csv` from here because it does not exist anymore, obviously (and thankfully we have tests that caught this).
I thought that it makes sense however to add the `File.read` extension that the Table library provides, since it is a replacement for that functionality (heavily extending it).

If you think adding it here was wrong please let me know, I will fix it in a next PR, as I'd like to proceed with this one to not stall other PRs depending on it, given it's a rather minor change we don't see anyway.

Reply:
That's absolutely fine - thought `File.read` was already there but clearly not.